lib/ukrust/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+license-file = "../../LICENSE.txt"
```

Comment:
Which license file does this reference? Preferably, we would specify `license` instead of `license-file`: https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields

Reply:
Good catch! I think unikraft is under the BSD 3 license, but not sure if there are changes. This is the main license file https://github.com/unikraft/unikraft/blob/staging/COPYING.md , I'll update the path to it.